azext_edge/tests/edge/support/create_bundle_int/test_arcagents_int.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+import pytest
+from knack.log import get_logger
+from azext_edge.edge.common import OpsServiceType
+from azext_edge.edge.providers.support.arcagents import ARC_AGENTS
+from .helpers import (
+    check_workload_resource_files,
+    get_bundle_path,
+    get_file_map,
+    run_bundle_command
+)
+
+logger = get_logger(__name__)
+
+AGENT_RESOURCE_PREFIXES = {
+    "cluster-identity-operator": "clusteridentityoperator",
+    "clusterconnect-agent": "clusterconnect-agent",
+    "config-agent": "config-agent",
+    "extension-events-collector": "extension-events-collector",
+    "extension-manager": "extension-manager",
+    "kube-aad-proxy": "kube-aad-proxy",
+    "cluster-metadata-operator": "cluster-metadata-operator",
+    "metrics-agent": "metrics-agent",
+    "resource-sync-agent": "resource-sync-agent"
+}
+
+
+@pytest.mark.parametrize("ops_service", OpsServiceType.list()[1:])
```

Comment:
if you aren't testing other services, is there a reason to include this? 
you can just have it bundle all (or bundle a smaller service if you want there to be less files)

Reply:
I actually wanted to test --arc with all services tho... is that too time consuming?

Reply:
it does increase the number of tests - if the arc resources are separate from the pods, there is no need here.

Reply:
ok will update